client/package.json

```diff
@@ -30,6 +30,8 @@
     "nextjs": "0.0.3",
     "react": "17.0.2",
     "react-dom": "17.0.2",
+    "styled-components": "^5.3.3",
```

Comment:
`styled-components` can be removed since we stopped using that after we migrated to using Chakra-UI